src/components/RedoIcon.tsx

```diff
@@ -9,7 +9,7 @@ const RedoIcon = ({ fill, size = 18, style = {}, cssRaw, animated, animationComp
       <svg
         xmlns='http://www.w3.org/2000/svg'
         viewBox='0 0 24 24'
-        style={{ ...style, width: '100%', height: '100%' }}
+        style={{ ...style, width: '100%', height: '100%', transform: 'scale(0.96, 0.96)' }}
```

Comment:
My only concern with scaling is that it will cause line widths no longer be uniform. All of the icons were custom designed to have the same stroke width.

Can you tell me a little about what the scaling is solving?

Reply:
Okay, thanks. They were all made by the same designer to the same specifications, so I'm not sure why some icons would need to be scaled up while others need to be scaled down. But since none of them are scaled more than 5–10%, I don't think the difference in stroke width will be noticeable. They look good to me across font sizes now, so it seems to be working.